src/fault_injector/fault_types/track_blocked_fault.py

```diff
@@ -0,0 +1,16 @@
+from src.component import Component
+from src.fault_injector.fault_types.fault import Fault, FaultConfig
+
+
+class TrackBlockedFault(Fault):
```

Comment:
Because `TrackBlockedFault` doesn't implement `resolve_fault` it's an abstract class again 😇 Just implement `resolve_fault` with `pass` and a comment 👍